todo_app/.env.test

```
@@ -0,0 +1,20 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+SECRET_KEY=test_secret_key
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+
+API_KEY=test_key_value
+API_TOKEN=test_token_value
+
+BOARD_ID=6514575b05d27488d6c36694
+TRELLO_TODO_LIST_ID=6514575cf3a727692b524b47
+DONE_LISTID=6514575c5e0f1594f3204a00
+DOING_LISTID=6514575c022eae157e8ca83b
```

Not that it matters since these aren't sensitive, but I often think better to use obviously fake values for your tests; it can be another sense-check that if you spot real-looking board ids in tests then we might have mixed up our test & real variables:
```suggestion
BOARD_ID=board_id
TRELLO_TODO_LIST_ID=todo_id
DONE_LISTID=done_id
DOING_LISTID=doing_id
```